rust/arrow/src/csv/reader.rs

```diff
@@ -395,8 +395,8 @@ pub struct ReaderBuilder {
 }
 
 impl Default for ReaderBuilder {
-    fn default() -> ReaderBuilder {
-        ReaderBuilder {
+    fn default() -> Self {
```

Comment:
Maybe it's a better idea to put this refactoring into another PR? It doesn't seem related with csv writer

Reply:
The changes are too small, but I can revert them. I wanted to make the reader consistent with the writer